app/controllers/api/v1/keywords_controller.rb

```diff
@@ -22,6 +22,18 @@ def create
         end
       end
 
+      def show
+        keyword = policy_scope(Keyword).find(params[:id])
+        options = { include: [:source] }
+
+        render json: KeywordDetailSerializer.new(keyword, options).serializable_hash.to_json
+      rescue ActiveRecord::RecordNotFound
+        render_errors(
+          details: [I18n.t('keyword.not_found')],
+          status: :not_found
+        )
```

Comment:
Can we handle this part globally? I mean we should not `recuse` on every `show` like this when the resource does not exist.

Reply:
@byhbt Do you mean to declare this `render_errors(` as a private variable inside this class?

> should not recuse on every show

How can we determined if the keyword exist for the particular user + keyword? The idea is to show only keyword that the user owned. 🤔